### ISSUE TITLE
Partitions string issue.

### ISSUE THREAD

**Issue by aouinizied** [NLeSC/dask-cassandra-loader#3]
Hi all, 

There is an issue with how partitions are written. In fact in loader (line 313) it is systematically mapped to float.
However, in case you have an str value partition it will never be correctly handled.

Best regards,

**Comment by romulogoncalves** [comment]
Indeed, that is an issue. For the moment we thought on only support integers and floats, but even in that case the documentation is incomplete.

We should also support strings and in the up-coming release it will be fixed.
